ui/app/components/app/wallet-view/wallet-view.component.js

```diff
@@ -0,0 +1,140 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import BalanceComponent from '../../ui/balance'
+import AddTokenButton from '../add-token-button'
+import AccountDetails from '../account-details'
+
+const { checksumAddress } = require('../../../helpers/utils/util')
+const TokenList = require('../token-list')
+const { ADD_TOKEN_ROUTE } = require('../../../helpers/constants/routes')
+
+export default class WalletView extends Component {
+  static contextTypes = {
+    t: PropTypes.func,
+    metricsEvent: PropTypes.func,
+  }
+
+  static defaultProps = {
+    responsiveDisplayClassname: '',
+    selectedAddress: null,
+    selectedAccount: null,
+    selectedTokenAddress: null,
+  }
+
+  static propTypes = {
+    selectedTokenAddress: PropTypes.string,
+    selectedAccount: PropTypes.object,
+    selectedAddress: PropTypes.any,
```

Comment:
This is always a string. And as mentioned in the other comment, should probably be required.